packages/modules/cra-template-desktop-viewer/template.json

```diff
@@ -0,0 +1,78 @@
+{
+  "package": {
+    "dependencies": {
+      "@bentley/backend-itwin-client": "^2.14.0",
+      "@bentley/bentleyjs-core": "^2.14.0",
+      "@bentley/build-tools": "^2.14.0",
+      "@bentley/context-registry-client": "^2.14.0",
+      "@bentley/ecschema-metadata": "^2.14.0",
+      "@bentley/electron-manager": "^2.14.0",
+      "@bentley/express-server": "^2.14.0",
+      "@bentley/frontend-authorization-client": "^2.14.0",
+      "@bentley/geometry-core": "^2.14.0",
+      "@bentley/icons-generic-webfont": "^1.0.15",
+      "@bentley/imodel-select-react": "^2.0.7",
+      "@bentley/imodelhub-client": "^2.14.0",
+      "@bentley/imodeljs-backend": "^2.14.0",
+      "@bentley/imodeljs-common": "^2.14.0",
+      "@bentley/imodeljs-frontend": "^2.14.0",
+      "@bentley/imodeljs-i18n": "^2.14.0",
+      "@bentley/imodeljs-markup": "^2.14.0",
+      "@bentley/imodeljs-quantity": "^2.14.0",
+      "@bentley/itwin-client": "^2.14.0",
+      "@bentley/orbitgt-core": "^2.14.0",
+      "@bentley/presentation-backend": "^2.14.0",
+      "@bentley/presentation-common": "^2.14.0",
+      "@bentley/presentation-components": "^2.14.0",
+      "@bentley/presentation-frontend": "^2.14.0",
+      "@bentley/product-settings-client": "^2.14.0",
+      "@bentley/rbac-client": "^2.14.0",
+      "@bentley/react-scripts": "3.4.11",
+      "@bentley/telemetry-client": "^2.14.0",
+      "@bentley/ui-abstract": "^2.14.0",
+      "@bentley/ui-components": "^2.14.0",
+      "@bentley/ui-core": "^2.14.0",
+      "@bentley/ui-framework": "^2.14.0",
+      "@bentley/ui-ninezone": "^2.14.0",
+      "@bentley/webgl-compatibility": "^2.14.0",
+      "@itwin/desktop-viewer-react": "^1.0.0",
+      "@reduxjs/toolkit": "^1.6.0",
+      "@types/electron-devtools-installer": "^2.2.0",
+      "@types/minimist": "^1.2.0",
+      "@types/react": "^16.9.0",
+      "@types/react-dom": "^16.9.0",
+      "@types/react-redux": "^7.1.9",
+      "cross-env": "^5.2.1",
+      "dotenv-flow": "^3.2.0",
+      "electron": "^11.1.0",
+      "electron-devtools-installer": "^2.2.3",
+      "minimist": "^1.2.5",
+      "npm-run-all": "^4.1.5",
+      "react": "16.13.1",
+      "react-dom": "16.13.1",
+      "react-redux": "^7.2.0",
+      "sass": "^1.29.0",
+      "typescript": "^3.7.2",
+      "webpack": "4.42.0"
+    },
+    "scripts": {
+      "build:backend": "tsc -p tsconfig.backend.json",
+      "build:frontend": "pseudolocalize && react-scripts build",
+      "electron": "electron lib/backend/main.js",
+      "electron:debug": "cross-env NODE_ENV=development electron lib/backend/main.js",
+      "start": "npm run build:backend && run-p \"start:frontend\" \"electron:debug\"",
+      "start:frontend": "react-scripts start",
+      "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./public/locales/en-PSEUDO"
+    },
+    "browserslist": [
+      "last 4 chrome version",
```

Comment:
for Desktop, lets default this to "Electron 10".

Reply:
updated